.github/workflows/go.yml

```diff
@@ -1,5 +1,12 @@
 name: Go
-on: [push, pull_request]
+on: 
+  push: 
+    branches: 
+      - master
```

Comment:
Actually, that was my mistake leaving my change in. I just didn't want it trying to build in my fork, since I have workflows turned on. I'll revert it.